src/embedding_scvi/_components.py

```diff
@@ -8,6 +8,38 @@
 from torch import nn
 
 
+class ConditionalBatchNorm2d(nn.Module):
+    def __init__(self, num_features, num_classes, momentum, eps):
+        super().__init__()
+        self.num_features = num_features
+        self.bn = nn.BatchNorm1d(self.num_features, momentum=momentum, eps=eps, affine=False)
+        self.embed = nn.Embedding(num_classes, self.num_features * 2)
+        self.embed.weight.data[:, :self.num_features].normal_(1, 0.02)  # Initialise scale at N(1, 0.02)
+        self.embed.weight.data[:, self.num_features:].zero_()  # Initialise bias at 0
```

Comment:
Might be a bit clearer if we instead have two `Embedding`s, one for scale and one for bias